### PR TITLE
#1 updated my data base

### PR DESCRIPTION
**Pull request by osadolortech**

### CODE REVIEW

File: config/database.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from http import client
+from pymongo import MongoClient
+
+client = MongoClient("mongodb+srv://FinalBlog :Computer333@cluster0.ve2nnja.mongodb.net/mainblog?retryWrites=true&w=majority")
```
Comment (iamr0b0tx) [inline config/database.py]: use env variables

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: config/database.py (hunk 1 above)
Comment (osadolortech) [reply config/database.py]: changes done sir

Comment (osadolortech) [reply config/database.py]: changes done sir

Comment (osadolortech) [reply config/database.py]: changes done sir

**Review by osadolortech — COMMENTED** [review]
changes made

File: config/database.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from http import client
+from os import getenv
+from pymongo import MongoClient
+
+username = getenv("DB_USERNAME")
+password = getenv("DB_BASE")
+
+client = MongoClient("mongodb+srv://username:password@cluster0.ve2nnja.mongodb.net/mainblog?retryWrites=true&w=majority")
```
Comment (iamr0b0tx) [inline config/database.py]: why not just make the entire database URL an ENV Variable

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: config/database.py (hunk 2 above)
Comment (osadolortech) [reply config/database.py]: please sir check if the new update is fine now

**Pull request merged**